tests/requirements/base.txt

```diff
@@ -10,3 +10,5 @@ pytz > dev
 selenium
 sqlparse
 tblib
+asgiref
+daphne
```

Comment:
If these are going in setup.cfg, then I guess I should modify the Jenkins build script to `pip install -e path/to/checkout` so we don't need to duplicate them here. Agreed?